### ISSUE TITLE
Switch to using official VOS images

### ISSUE THREAD

**Issue by amoeba** [DataONEorg/slinky#34]
Virutoso now (since 2017?) has official images. We should use those I think: https://hub.docker.com/r/openlink/virtuoso-opensource-7/.

**Comment by amoeba** [comment]
I swapped the image the test suite's docker-compose uses from the [tenforce](https://hub.docker.com/r/tenforce/virtuoso/) image to the [official one](https://hub.docker.com/r/openlink/virtuoso-opensource-7) in 72b00fae54e40183e9bcf360bfc9de1ca1bd0f8e.

I wasn't expecting any compatibility issues and there were none: The test suite runs fine under the new image. I did run into one fun thing that changed: The old image let you turn on SPARQL UPDATE via environment varariable but the new image doesn't have the same facility. I settled on creating a container the runs once that waits on Virtuoso to start up and then turns on SPARQL UPDATE via ISQL. 

Part of the reason why such a complicated technique is needed is because the VOS image doesn't actually have a fully installed VOS instance that's ready to start and initial configuration is done on first run. I wanted to avoid a second container so I tried what the [tenforce](https://github.com/tenforce/docker-virtuoso) image did which is basically (1) start VOS in the background (2) run the SQL we need and (3) foreground the VOS. I couldn't get that same technique to work with the official images.

I think we'll need to deal with this again for the production deployment @ThomasThelen. FYI.
